@xen-orchestra/web-core/lib/components/LegendTitle.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <!-- Legend title v1.0.1 -->
+  <span v-if="title" class="legend-title typo c3-semi-bold"
+    >{{ title }} <UiIcon :icon="hasIcon !== false ? faFaceSmile : undefined" color="info" class="ui-icon"
+  /></span>
+</template>
+<script setup lang="ts">
+import { faFaceSmile } from '@fortawesome/free-solid-svg-icons'
+import UiIcon from '@core/components/icon/UiIcon.vue'
+interface Props {
+  title: string
+  hasIcon: boolean
+}
+defineProps<Props>()
+</script>
+<style lang="postcss" scoped>
+.legend-title {
+  color: var(--color-grey-200);
+  display: flex;
+  gap: 0.8rem;
+  align-items: center;
+
+  & .ui-icon {
+    font-size: 1.2rem;
+  }
+}
+</style>
```

Comment:
- Move the component version to the top of the file
- As discussed, component version should only contain `x.y` (`z` is for Clémence internal versioning on Figma)
- No need to add `v-if="title"` since it's a required prop
- No need to do an explicit comparison on a boolean `hasIcon !== false ? ...` => `hasIcon ? ...`
- A `div` is more appropriate than a `span` in this case
- The `icon` must be customizable
- According to DS, the icon should have a size of `1.6rem`

```suggestion
<!-- v1.0 -->
<template>
  <div class="legend-title typo c3-semi-bold">
    {{ title }}
    <UiIcon :icon color="info" />
  </div>
</template>

<script setup lang="ts">
import UiIcon from '@core/components/icon/UiIcon.vue'

defineProps<{
  title: string
  icon?: IconDefinition
}>()
</script>

<style lang="postcss" scoped>
.legend-title {
  color: var(--color-grey-200);
  display: flex;
  gap: 0.8rem;
  align-items: center;
}

.ui-icon {
  font-size: 1.6rem;
}
</style>
```